workitem.go

```diff
@@ -126,10 +126,18 @@ func (c *WorkitemController) Update(ctx *app.UpdateWorkitemContext) error {
 func (c *WorkitemController) Reorder(ctx *app.ReorderWorkitemContext) error {
 	return application.Transactional(c.db, func(appl application.Application) error {
 		var dataArray []*app.WorkItem2
-		if ctx.Payload == nil || ctx.Payload.Data == nil {
+		if ctx.Payload == nil || ctx.Payload.Data == nil || ctx.Payload.Position == nil {
 			return jsonapi.JSONErrorResponse(ctx, errors.NewBadParameterError("missing payload element in request", nil))
 		}
 
+		if ctx.Payload.Position.ID == "" {
```

Comment:
Please see my comment above about the `a.Enum("above", "below")` or `a.MinLength(1)`. By applying any of the solutions you can get rid of this check.